package.json

```diff
@@ -27,7 +27,7 @@
         "theme": "light"
     },
     "engines": {
-        "vscode": "^1.68.0-insider"
+        "vscode": "^1.68.0-insiders"
```

Comment:
this was correct before

Reply:
Oh I typed it in wrong when I made the cherry-pick. In the release branch it's removed.